Testnet/DAOContract/DAOContract/DAOContract.cs

```diff
@@ -188,6 +188,18 @@ public void Deposit()
         Log(new FundRaisedLog { Sender = Message.Sender, Amount = Message.Value });
     }
 
+    public void UpdateMinQuorum(uint minQuorum)
+    {
+        EnsureOwnerOnly();
+        this.MinQuorum = minQuorum;
```

Comment:
You can automate this is what I was saying. If you keep track of number of whitelisted addresses then 
`MinQuorum = WhitelistedAddressesCount / 2 + 1` This calculation will always return 50% + 1 and works for any number over 2. 

Up to you and the team if you want to update this manually every single time an address is whitelisted or blacklisted, but this calculation enforces ~51% approval automatically at all times if implemented 👍  up to you guys.

Reply:
can resolve this too

Reply:
You commented on outdated version of the file

Reply:
mark this comment as resolved is what I meant. I don't have permissions